src/Sarif.Viewer.VisualStudio.Core/Options/SarifViewerOptions.cs

```diff
@@ -30,6 +32,8 @@ private SarifViewerOption(AsyncPackage package)
 
         public bool ShouldMonitorSarifFolder => this.optionPage?.MonitorSarifFolder ?? this.shouldMonitorSarifFolderDefaultValue;
 
+        public bool GhasIntegrationEnabled => this.optionPage?.GhasIntegrationEnabled ?? this.ghasIntegrationEnabledDefaultValue;
```

Comment:
>GhasIntegrationEnabled [](http://example.com/codeflow?start=20&length=22)

this changeset adds a checkbox, 
don't see code change how this checkbox is used, 
it is not included in this changeset right